### DISCUSSION TITLE
Is domain name mandatory for installation ? Is it possible to proceed without having that?

### DISCUSSION THREAD

**Discussion by vipinvijay1986** [egovernments/Digit-Core#43] [DIGIT Platform]
Dear team, 
Is it mandatory to have a domain name for full installation? and if it's not then what is the way to proceed with it. Can I pass IP detail for the same? 

Reference page link 
https://docs.digit.org/urban/platform/setup-digit/install-on-cloud/on-aws/5.-prepare-deployment-config

**Comment by gajenc** [comment]
Yes, you can use IP. domain name is essentially for the application to access. If you are using IP instead of a URL, it's your choice. The full install still works. But I would recommend using the DNS.
